src/content/reference/runtime-apis/kv.md

```diff
@@ -30,15 +30,19 @@ await NAMESPACE.put(key, value)
 
 This method returns a `Promise` that you should `await` on in order to verify a successful update.
 
-You can also [write key-value pairs from the command line with Wrangler](/reference/wrangler-cli/kv_commands/#kv-key).
+You can also [write key-value pairs from the command line with
+Wrangler](/reference/wrangler-cli/commands#kv:key).
```

Comment:
```suggestion
Wrangler](/reference/wrangler-cli/commands#kvkey).
```

Reply:
How does this syntax work? The heading looks like this #### `kv:key`

Reply:
I just navigated to the page and clicked on the headers to see what fragment URL they're using.

Long story short, we're using `remark-slug` to generate these slugs automatically, and it's using "GitHub's algorithm" for generating slugs from headings: https://github.com/remarkjs/remark-slug